premailer/premailer.py

```diff
@@ -417,6 +417,12 @@ def transform(self, pretty_print=True, **kwargs):
                         parent.attrib[attr].startswith('cid:')
                     ):
                         continue
+                    if parent.attrib[attr].startswith("//"):
+                        try:
+                            protocol = self.base_url.split("//")[0]
+                        except IndexError:
+                            protocol = "http"
```

Comment:
Hmm, I'm not sure... If we don't touch it, we're letting mail client handle "//" links and they don't really do it well (as far as I can tell, at least OSX Mail and Thunderbird don't like it. Gmail does it well but it's in a browser so it's cheating...). We're potentially talking about the combination of :

 - Using ```"mysite.com"``` as base url
 - Having ```<img src="//othersite.com/static/img.jpg">```

that would render to ```<img src="//othersite.com/static/img.jpg">``` which mail client would not understand, I'm not convinced it's the best choice... Or maybe I overlooked something ?

(Note that if the base url itself is defined as ```"//mysite.com"``` that would make links defined the same way with my implementation (but then you get what you asked for).)

Maybe a solution would be to just remove ```//``` so we would get simply ```<img src="mysite.com/static/img.jpg">``` that mail clients will probably interpret as http